quinn/src/endpoint.rs

```diff
@@ -183,10 +185,9 @@ impl Endpoint {
             addr
         };
         let (ch, conn) = endpoint.inner.connect(config, addr, server_name)?;
-        let udp_state = endpoint.udp_state.clone();
-        Ok(endpoint
-            .connections
-            .insert(ch, conn, udp_state, self.runtime.clone()))
+        let dirty = endpoint.dirty_send.clone();
```

Comment:
would it be worthwhile to do this in the `Connection` context/constructor, which also triggers the other wakeups?   As far as I understand the dirty sender gets forwarded there anyway.

Reply:
That does seem a little cleaner, especially since it deduplicates the equivalent send for incoming connections.